code/src/terrat_vcs_github_comment/terrat_vcs_github_comment.mli

```diff
@@ -0,0 +1,90 @@
+module Api = Terrat_vcs_api_github
+module Scope = Terrat_scope.Scope
+
+module Sql : sig
```

Comment:
Do we need to expose the Sql module to the outside world? I would have imagined it's private to this module.

Reply:
done